### ISSUE TITLE
Thread-safe problem while inserting

### ISSUE THREAD

**Issue by gamtruliar** [dieselpoint/norm#80]
java.sql.SQLException: SQL String cannot be NULL and the error stack is: com.dieselpoint.norm.DbException: java.sql.SQLException: SQL String cannot be NULL

**Comment by ccleve** [comment]
Yup, I can see how this can happen. I just added a "synchronized" keyword to the method and pushed it to master. Version bumped to 1.0.6. Can you test it to see if it fixes the problem?

**Comment by ccleve** [comment]
@gamtruliar Did the change fix your problem?
